src/styles/main.scss

```diff
@@ -1,3 +1,59 @@
+$date-size: 100px;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
```

Comment:
make variable $gap-size for gap and use in loop which counts margin-left for first day